### PR TITLE
Investigate differing file permissions on Travis

### PR DESCRIPTION
**Pull request by pmeinhardt**

### CODE REVIEW

**Comment by pmeinhardt** [comment]
@klappradla, this works around the Travis CI permissions issue… 🤔 
it's just credo that's still complaining now (as in the target branch from #96) 👋

**Comment by pmeinhardt** [comment]
I'm still not sure why permissions are different on Travis…
`git ls-tree HEAD test/fixtures/*` shows the correct permissions locally…

**Comment by klappradla** [comment]
Fixed 👮‍♀️ things on the target branch....

Can we merge this PR into that branch...?

File: .travis.yml [hunk 1]
```diff
@@ -17,6 +17,9 @@ matrix:
       otp_release: 20.2
       services: docker
 
+before_script:
+  - chmod -R g-w test/fixtures # y u no work as expected travis?
```
Comment (klappradla) [inline .travis.yml]: I thought we already tried this yesterday... 🤔 But 🎉 if it works

**Comment by pmeinhardt** [comment]
I guess so 😀

**Pull request merged**